pkg/driver/vncdriver/vnc/encoding.go

```diff
@@ -167,14 +193,19 @@ func (ze *ZlibEncoding) Read(c *ClientConn, rect *Rectangle, r io.Reader) (Encod
 				color.R = uint16((rawPixel >> c.PixelFormat.RedShift) & uint32(c.PixelFormat.RedMax))
 				color.G = uint16((rawPixel >> c.PixelFormat.GreenShift) & uint32(c.PixelFormat.GreenMax))
 				color.B = uint16((rawPixel >> c.PixelFormat.BlueShift) & uint32(c.PixelFormat.BlueMax))
+				if c.PixelFormat.BPP == 16 {
+					color.B = color.B<<3 | color.B>>2
+					color.G = color.G<<2 | color.G>>4
```

Comment:
Is it intended that it differs from
https://github.com/pion/mediadevices/pull/389/files#diff-aa6d77c86375eb2033317eb0e9c570eb8af96eaf52db50a300c10f9667992d4aR87
> ```
> 					color.G = color.G<<2 | color.G>>2
> ```
?